sns/parity_math.c

```diff
@@ -463,6 +490,9 @@ M0_INTERNAL int m0_parity_math_init(struct m0_parity_math *math,
 		ec_init_tables(data_count, parity_count,
 			       &math->pmi_encode_matrix[data_count * data_count],
 			       math->pmi_encode_tbls);
+
+		M0_LOG(M0_DEBUG, "use Intel ISA for parity calculation.");
```

Comment:
You need to take this log at the start of this else statement so that if any of the error paths are taken we would still know this was the Intel ISA path. In the current code if we have an error during allocation we would not know we had entered this path.
I will approve this PR but make this small change and no need to wait for approvals for this small change.